specification/Metadata/README.md

```diff
@@ -47,26 +48,34 @@ Draft
     - [Semantic](#semantic)
     - [Type](#type)
     - [Component Type](#component-type)
+    - [Enum Type](#enum-type)
+    - [Arrays](#arrays)
     - [Normalized Values](#normalized-values)
+    - [Offset and Scale](#offset-and-scale)
     - [Minimum and Maximum Values](#minimum-and-maximum-values)
-    - [Required Properties and No Data Values](#required-properties-and-no-data-values)
+    - [Required Properties, No Data Values, and Default Values](#required-properties-no-data-values-and-default-values)
+    - [Property Values Structure](#property-values-structure)
 - [Storage Formats](#storage-formats)
   - [Overview](#overview-2)
   - [Binary Table Format](#binary-table-format)
     - [Overview](#overview-3)
-    - [Numbers](#numbers)
+    - [Scalars](#scalars)
+    - [Vectors](#vectors)
+    - [Matrices](#matrices)
     - [Booleans](#booleans)
     - [Strings](#strings)
     - [Enums](#enums-1)
-    - [Arrays](#arrays)
+    - [Fixed-Length Arrays](#fixed-length-arrays)
+    - [Variable-Length Arrays](#variable-length-arrays)
   - [JSON Format](#json-format)
     - [Overview](#overview-4)
-    - [Numbers](#numbers-1)
+    - [Scalars](#scalars-1)
+    - [Vectors](#vectors-1)
+  - [Matrices](#matrices-1)
```

Comment:
Different indentation - intentional? (Also different number of `#` in linked-to section)

Reply:
Ah, that was not intentional. Fixed in https://github.com/CesiumGS/3d-tiles/pull/630/commits/f49d52e244c512fb5a595e994f69d1c90f5eafea